packages/shared/src/components/fields/Radio.tsx

```diff
@@ -46,7 +46,7 @@ export function Radio({
           disabled={option.disabled}
           checked={value === option.value}
           onChange={() => onChange(option.value)}
-          className={classNames('my-0.5 truncate', className.content)}
+          className={classNames('truncate', className.content)}
```

Comment:
does removing this my-0.5 mess with the design of any other usage of radio?

Reply:
As far as I understand, we removed the `-my-0.5` on the container div and also removed `my-0.5` here. 
The `my-0.5` was used here to add gap between elements, which is now handled by `gap-1` on the container itself instead. 

The negative margin was used on the container so that the first item did not have unnecessary y margin. There should be no effect of this change as far as I can see.

Reply:
Most of the time, we now use `gap-` to apply the correct spacing in-between elements, which is what the margin is doing here. This should be safe.